docs: new readme and docs (#521)
graphprotocol/indexer-rs: docs/Routes.md

| `/cost`                 | Provides access to the Cost Model GraphQL API.                                               |

## Subgraph Routes

| Route                                | Description                                                                                  |
|--------------------------------------|----------------------------------------------------------------------------------------------|
| `/subgraph/health/:id`               | Retrieves the health state of a specified subgraph using its ID.                             |
| `/subgraphs/id/:id`                  | Routes a query to a specific subgraph using its ID. Requires a receipt or valid token.       |

## Node Status Route

| Route                   | Description                                                                                  |
|-------------------------|----------------------------------------------------------------------------------------------|
| `/status`               | Routes requests to the graph-node status API.                                                |

---

## Note

> If this documentation is outdated or incomplete, you can view the complete and up-to-date list of routes in the source code:  

Reviewer: ```suggestion
You can always view the latest complete and up-to-date list of routes in the source code:  
```
